src/taskgraph/run-task/fetch-content

```diff
@@ -39,6 +39,11 @@ try:
 except ImportError:
     certifi = None
 
+try:
+    import taskcluster
+except ImportError:
+    taskcluster = None
```

Comment:
Nice idea, but in this case I think the benefits of using the TC client probably don't outweigh the complexity of supporting two ways of interacting with the API. So I think we should leave this file alone